### PR TITLE
Add Aria breadcrumbs

### PR DESCRIPTION
**Pull request by ayumihamsaki**
Related to here: https://github.com/octobercms/october/issues/4454

### CODE REVIEW

**Comment by ayumihamsaki** [comment]
Finished.

File: modules/system/assets/ui/docs/breadcrumb.md [hunk 1]
```diff
@@ -4,10 +4,12 @@ Display a breadcrumb on the page.
 
     <div style="padding: 20px">
         <div class="control-breadcrumb">
-            <ul>
-                <li><a href="#">Dash Board</a></li>
-                <li><a href="#">Blog Posts</a></li>
-                <li>Edit Post</li>
-            </ul>
+            <nav aria-label="Breadcrumb">
```
Comment (w20k) [inline modules/system/assets/ui/docs/breadcrumb.md]: @ayumihamsaki why do you remove `control-breadcrumb` class?

Comment (ayumihamsaki) [reply modules/system/assets/ui/docs/breadcrumb.md]: @w20k I didn't it on line 6 ?

Comment (w20k) [reply modules/system/assets/ui/docs/breadcrumb.md]: Sorry, my bad, morning sleepy eyes :)

Comment (ayumihamsaki) [reply modules/system/assets/ui/docs/breadcrumb.md]: haha, I thought I completely missed something!

**Comment by w20k** [comment]
@ayumihamsaki is the current link not selected at all (focused?)?

Screenshot 2019-08-07 at 16 11 18

**Comment by ayumihamsaki** [comment]
@w20k 

Over the years I have done a lot of research into that question!

1. According to the W3c spec on breadcrumbs you should NOT show a link to the current web page.

2. However, Google and Bing both have examples showing a link for the current web page. A few years ago I contacted both of them and told them in great detail their examples are wrong according to the spec.

3. Schema markup was created a few years later and they had mixed examples. At first they didn't have the current web page marked up and then the examples at present do have the current web page marked up.

4. Bad SEO practises going back 10 years ago it was a hack to link the current web page to increase the current web page Page Rank, something called: PageRank sculpting.

5. Going full circle and into 2019, WAI-ARIA spec with regards to breadcrumbs says to NOT markup the links using `role="link"` and just adding `aria-current="page"` to the current web page.

### So what does that all mean?

The reason I given a long answer is because the screenshot you showed is correct and not correct lol. For years I would have said yes don't add a link to the current web page according to the HTML spec set out by W3c and HTML Living Standard org.

But everything conflicts their standards and says to add a link to the current web page.

However, you have to ask a simple question and one I asked for many years!

If someone is on the current web page - why would they need to navigate to the same web page when they are on that web page? It doesn't make any sense.

So with all that info, my answer would be: No - I don't think you need to add the current link focused and the WAI-ARIA spec also says no. But all the other spec's conflict this.

**Comment by ayumihamsaki** [comment]
Closed this and moved to here: https://github.com/octobercms/october/pull/4531

**Pull request closed without merging**